src/components/SearchBar.js

```diff
@@ -1,14 +1,19 @@
-import React, { Component } from 'react'
-import ResultsList from './containers/ResultsList'
-import search from '../js/search'
```

Comment:
How about I leave the code as it is, but just close this PR request for now, and reopen it once the functionality is added? Because I think a lot of progress was made as far as the UI goes, and in it's current state, it's not too far from being fully functional (we probably only need 5-15 more lines of code to make it functional).